pallets/xst/src/lib.rs

```diff
@@ -1005,6 +1028,11 @@ impl<T: Config> Pallet<T> {
             )),
         }
     }
+
+    fn disable_synthetic_asset_unchecked(synthetic_asset: AssetIdOf<T>) {
+        EnabledSynthetics::<T>::remove(synthetic_asset);
```

Comment:
Now you can also disable XSTPool source for the trading pair by `trading_pair::disable_source_for_trading_pair`